README.md

```diff
@@ -2,23 +2,22 @@
 
 ## Problem Statement
 
-A key to collaborating with `git` is to keep discrete and individual lines of work
-isolated from each other. If you start work on a big feature and make a few commits
-that don't entirely finish the feature, you would not be able to commit these changes
-to the repository without potentially breaking the application, or having visibly unfinished
-work. You could, in theory, wait until the feature is done--but what if you notice there's a
-bug in the application for users that needs to be fixed immediately? Even if it's an easy
-fix if you made that commit with your other unfinished changes, you'd be pushing your
-half-finished and broken new feature. How can your work on the new feature be isolated, so
-that until it's done, you can deploy the commit that fixes the application? This can be done
-using a feature in `git` called a `branch`.
+Two golden practices in collaborating with `git` are:
+
+1. Keep the `master` _branch_ working at all times
```

Comment:
We use the word "work" a lot when talking about git branches. Here I thought for a second this was using "working" to mean "in progress," but I think it's meant to mean "functional." If that's the case, I'd suggest a different word choice here for maximum clarity.

Reply:
✔️